src/training/_hooks.py

```diff
@@ -119,7 +123,7 @@ def after_run(self, context, values):
       throughput = self._global_steps_completed * float(self._batch_size) / float(duration)
 
       logging.info(LogTrainingHook._MESSAGE_FORMAT,
-                   self._global_steps_completed, duration, throughput, loss_value)
+                   self._global_steps_completed, loss_value)
```

Comment:
oh, interesting point. As a smaller step, what if we prepend the word 'master' to LogTrainingHook?